extensions/amp-story/1.0/amp-story-bling-link.css

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-story-grid-layer amp-story-bling-link {
+  margin: 10px !important;
+  background: rgba(255, 255, 255, 1) !important;
+  border-radius: 50% !important;
+}
+
+amp-story-page[active] amp-story-bling-link {
+  animation: pulse 4s infinite !important;
```

Comment:
Animating on the `box-shadow` will trigger a layout + paint in every frame during the animation, using too many resources and potentially creating jank in the animation.
You can see what each CSS property trigger during an animation on https://csstriggers.com/

For this kind of animation, I think we want to have a circle that grows/shrinks through a `transform: scale(...)`. As for the timing function, we've been trying to follow the material guidelines, that'd recommend using `cubic-bezier(0.4, 0.0, 0.2, 1);` :)

Lmk if this makes sense or if I can help with that!

Reply:
This is really cool to know, thanks for the heads up. I think I have a working solution, PTAL! One note is that I'm creating more dom elements so after `buildCallback()` this is what the dom looks like 
```
<amp-story-bling-link id="blink-1" layout="fixed" width="90px" height="90px">
  <div class="i-amphtml-story-bling-link-pulse"></div>
  <div class=i-amphtml-story-bling-link-button>
    <i class="i-amphtml-story-bling-link-icon i-amphtml-story-bling-link-shopping-cart"></i>
  </div>              
</amp-story-bling-link>
```